tools/release.sh

```diff
@@ -58,5 +58,4 @@ if [ "$DRY_RUN" = 'true' ]; then
     git show
 else
     git push origin "$BRANCH"
-    jx start pipeline nuxeo/nuxeo-ai --branch "v$RELEASE_VERSION" || true
 fi
```

Comment:
small echo missing? `echo "$RELEASE_VERSION" >RELEASE_VERSION`

Reply:
Nice shot, you made me doubt :) It was actually already (and better) implemented in this repository: [line 36](https://github.com/nuxeo/nuxeo-ai/pull/250/files/1280a4e78e7a61886e353c21503f20d5e686e2ba#diff-b91eeb35e4e3920b1a96d2d03212320aR36).